pkg/client/didexchange/client.go

```diff
@@ -15,9 +15,11 @@ import (
 
 	"github.com/google/uuid"
 
-	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/model"
```

Comment:
What's the rationale for a `model` package?

Reply:
It contains DIDCommMsg and Destination. These are used handling the messages throughout the framework. ie, pass to handle from input, pass in event messages.

May be messages would make more sense ? Any suggestions ?